integration-libs/s4-service/order/components/cancel-service-order/cancel-service-order.component.html

```diff
@@ -0,0 +1,42 @@
+<form [formGroup]="form" (ngSubmit)="cancelServiceOrder()">
+  <div class="form-group">
+    <label for="reasonTextarea">{{
+      'cancelService.cancelReason' | cxTranslate
+    }}</label>
+    <textarea
+      id="reasonTextarea"
+      class="form-control"
+      formControlName="cancelReason"
+      rows="5"
+      (input)="updateCharacterLeft()"
+    ></textarea>
+    <div class="characters-left">
+      <span>{{
+        'cancelService.charactersLeft' | cxTranslate: { count: characterLeft }
+      }}</span>
+    </div>
+  </div>
+
+  <div class="row mt-5">
+    <div class="col-xs-12 col-md-6 col-lg-6">
+      <button
+        type="button"
+        class="btn btn-secondary btn-block back-button"
+        *ngIf="order$ | async as order"
+        [routerLink]="
+          {
+            cxRoute: 'orderDetails',
+            params: order
```

Comment:
Shouldnt this be order.code ? Just a question

Reply:
orderDetails route expects a detailed order object, so need to pass order